tests/testthat/test-parseTheme.R

```diff
@@ -0,0 +1,36 @@
+describe("parseTheme", {
+  it("should read the theme json and return a list", {
+    jsonString <- "{\n\"themePrimary\":\"black\"}"
```

Comment:
Kudos for adding tests! Nitpicks:
1. You can use single quotes (`'`) for strings to avoid escaping double quotes (`"`).
2. Perhaps write `it("reads the theme...")` instead of `it("should read the theme...")` avoid "should" everywhere :wink:.

Reply:
Done ✔️